### PR TITLE
[PPP] Update ppp draft

### PR DESCRIPTION
**Pull request by jaredlhf**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -70,13 +70,19 @@ The result of executing the input command will be displayed the lower-most box.
 
 ## Features
 
+### Command Format
+
+**Format:** `command_word Prefix/PARAMETER`
+
+* `command_word` is a word at the start of the command to specify the action to be done. <br> e.g. in `add n/NAME`, `add` is the command word.
+* `PARAMETER` are word(s) in `UPPER_CASE` to be supplied by the user.<br> e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`
+* `Prefix` is a letter before `PARAMETER` to denote the information supplied as parameter. <br>e.g. in `add n/NAME`, `n/` is a prefix to indicate that NAME is being supplied.
```
Comment (VimuthM) [inline docs/UserGuide.md:79]: Prefix can also be 2 letters. Better to modify this

File: docs/UserGuide.md [hunk 2]
```diff
@@ -108,15 +114,20 @@ Adds a resident and their information to the application.
...
-**:information_source: Parameter constraints:**<br>
-* `NAME` has to be unique.
-* `PHONE` is at least 3 digits long.
-* `ROOM` is of the form `BLOCK` + `LEVEL` + `NUMBER`, where `BLOCK` is an alphabet, `LEVEL` and `NUMBER` make up 3 digits. e.g. `A100`
-* `VACCINATION_STATUS` can be `T` or `F` (case insensitive).
-* `FACULTY` has to be a single alphabetical word.
-* The dates inputted for `LAST_FET_DATE` and `LAST_COLLECTION_DATE` has to be in dd-mm-yyyy format.
+**:information_source: Note:**<br>
+
+**ROOM** is currently modelled after an existing hall in the National University of Singapore, options for more flexibility will be made in the future.
```
Comment (VimuthM) [inline docs/UserGuide.md:130]: I dont think we should explicitly say flexibility will be provided in the future. What do you think?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -282,10 +293,18 @@ Example csv:
 
 #### Exporting residents' emails : `export`
 
-Exports the emails of all the residents in the last filtered list to a filename of choice within the `data/exports/` folder.
-As they are comma-separated, a quick copy-paste allows you to send mass emails using modern email clients.
+Exports the emails of all the residents in the last filtered list to a csv file within the `data/exports/` folder.
+A quick copy-paste of the email addresses from the csv file allows you to send mass emails using modern email clients.
 
-Format: `export FILE_NAME`
+Format: `export CSV_NAME`
```
Comment (VimuthM) [inline docs/UserGuide.md:299]: This parameter name change will need to be reflected in the codebase. Also note command suggestion has a hardcoded  snippet for this as well.

**Review by VimuthM — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (jaredlhf) [reply docs/UserGuide.md:79]: Good catch, will modify

File: docs/UserGuide.md (hunk 3 above)
Comment (jaredlhf) [reply docs/UserGuide.md:299]: Good point, will leave it as FILE_NAME

File: docs/UserGuide.md (hunk 2 above)
Comment (jaredlhf) [reply docs/UserGuide.md]: I've removed the part on "options for more flexibility will be made in the future."

**Review by jaredlhf — COMMENTED** [review]
There was an overlapping commit made on this branch, all ug changes have been relocated to pr #254.

**Review by VimuthM — APPROVED** [review]

**Pull request merged**